### PR TITLE
Add support for tags

### PR DESCRIPTION
**Pull request by himynameisjonas**
Make it possible to add tags to mails. These tags can later be seen/filtered by on the Flowmailer web.

#### Example where the message where tagged with `hello_world`
Screenshot 2021-08-11 at 09 55 51

### CODE REVIEW

File: spec/mail_converter_spec.rb [hunk 1]
```diff
@@ -31,6 +31,32 @@
           mimedata: "RGF0ZTogRnJpLCAwMSBKYW4gMjAxMCAxMjowMDowMCArMDAwMA0KRnJvbTog\nbm8tcmVwbHlAZXhhbXBsZS5jb20NClRvOiBqb2huLmRvZUBleGFtcGxlLmNv\nbQ0KTWVzc2FnZS1JRDogMTMzNw0KTWltZS1WZXJzaW9uOiAxLjANCkNvbnRl\nbnQtVHlwZTogdGV4dC9wbGFpbg0KQ29udGVudC1UcmFuc2Zlci1FbmNvZGlu\nZzogN2JpdA0KDQo=\n"
         )
       end
+
+      describe "tags" do
```
Comment (frysch) [inline spec/mail_converter_spec.rb:35]: Any idea to test with tags being an empty array or nil as well?

**Review by frysch — APPROVED** [review]
Looks sweet!

**Review by bjonord — CHANGES_REQUESTED** [review]
Having not seen this Gem before, I was a bit confused why we were creating a Dummy application inside the repository.

Think we could extract the creation of the dummy app in a separate commit and type up a short description of why it is required, for future developers. 😄

**Comment by himynameisjonas** [comment]
> Having not seen this Gem before, I was a bit confused why we were creating a Dummy application inside the repository.
> 
> Think we could extract the creation of the dummy app in a separate commit and type up a short description of why it is required, for future developers. 😄

I've pushed an update now with separate commits 👍

**Review by bjonord — APPROVED** [review]
Nice! 👍🏼

File: spec/mail_converter_spec.rb (hunk 1 above)
Comment (himynameisjonas) [reply spec/mail_converter_spec.rb:35]: yes and I've pushed a commit for just that now 😄

**Pull request merged**